commands/serverless.js

```diff
@@ -10,6 +10,8 @@ module.exports = function (input, flags, logger, options) {
   const cwd = options.cwd
   const out = flags.out
   const env = flags.env
+  const vpcSecurityGroups = flags.vpcSecurityGroups
+  const vpcSubnets = flags.vpcSubnets
```

Comment:
I think you may have made incorrect assumptions about how meow / minimist treats strings that are separated by a comma-followed-by-a-space

This PR handles comma-separated-without-spaces:

`node ./bin/index.js serverless --vpc-subnets no,space,just,commas`

but it won't have the expected result if there is a space after the comma:

`node ./bin/index.serverless --vpc-subnets space, after, each, comma`

Once spaces are involved, only `"space"` will be found in `cli.flags.vpcSubnets` whilst `"after,"`,  `"each,"`, and `"comma"` will be in `cli.input`

We might need another strategy for this

[minimist](https://github.com/substack/minimist) (which is what meow uses internally) has a `--` option, which if enabled will cause CLI argument parsing to stop after `--` is encountered. This could be useful for certain approaches.

Maybe something like ava, which internally encodes arguments into a single JSON string to pass to child processes: https://github.com/avajs/ava/pull/319 ?